docs/pages/cloud/architecture.mdx

```diff
@@ -47,6 +47,25 @@ The Teleport [proxy service](https://goteleport.com/docs/architecture/proxy/) ca
 
 The multi-region option is currently opt-in by default. Once you have an account, please reach out to your account manager, customer success engineer, or support@goteleport.com. A future update will expand the region availability and make all regions available by default.
 
+## Releases
+
+Teleport Cloud customers are typically running the latest release of the
+Teleport Auth and Proxy service.
+
+Teleport Cloud performs patch upgrades on a weekly basis and major upgrades on
+a quarterly basis. Teleport Cloud waits for the first minor release before a
+major upgrade (for example, 10.1 instead of 10.0). This typically occurs 3-4
+weeks after a major release.
+
+Patch releases are fully backward compatible and require no actions by the
+customer.
+
+Major releases do require customers to upgrade all agents within 3 months of
```

Comment:
```suggestion
Major releases require customers to upgrade all `teleport` and `tbot` instances within your cluster within 3 months of
```
Or some other alternative to "agent"—in the past, we've had some confusion from docs readers around the term "agent" and whether you need to deploy a Teleport binary on the same host as every resource you want to configure access to. I'm not sure if you would need to upgrade `tbot`.

Reply:
Updated the wording to mention all binaries the customer runs. What do you think?